packages/wp-story-editor/src/components/documentPane/index.js

```diff
@@ -44,4 +46,32 @@ function DocumentPane() {
   );
 }
 
+export function PublishModalDocumentPane() {
```

Comment:
@littlemilkstudio  thanks for this idea, even though it makes this file a bit longer, i think this makes way more sense.